modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/tracefeatureextraction/v0.1.0.json

```diff
@@ -0,0 +1,83 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/tracefeatureextraction/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:TraceFeatureExtractionShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:TraceFeatureExtraction",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Trace collection",
+              "description": "Used trace collections.",
+              "class": "nsg:TraceCollection",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
```

Comment:
The prov:used is used for two property shapes with different class targets. Use sh:qualifiedValueShape as for prov:generated property shapes below?

Reply:
agreed.